platform/default/mbgl/storage/offline_database.cpp

```diff
@@ -49,7 +49,8 @@ void OfflineDatabase::ensureSchema() {
             case 1: break; // cache-only database; ok to delete
             case 2: migrateToVersion3(); // fall through
             case 3: migrateToVersion4(); // fall through
```

Comment:
Thinking out loud: we could remove `OfflineDatabase::migrateToVersion4`, and have `case 3` fall through to `migrateToVersion5()` without doing anything itself. But then we'd have two types of databases in the wild: those that have been switched temporarily to WAL and back, and those that haven't. There _probably_ isn't any interesting differences between those two types, but why introduce the variable?

OTOH, what if switching to WAL, even temporarily, can causes an issue in some corner case? Then we _would_ be better off avoiding the migration through v4 when starting with v3.

I don't see a strong argument either way, so I'm fine with keeping this as is here.